README.md

```diff
@@ -35,7 +38,10 @@ Install with `npm install @octokit/core octokit-plugin-unique-issue`. Optionally
 
 ```js
 const { Octokit } = require("@octokit/core");
-const { uniqueIssue } = require("octokit-plugin-unique-issue");
+const {
+  uniqueIssue,
+  composeUniqueIssue,
```

Comment:
curious about opinions on naming. `composeUniqueIssue` aligns well with the `uniqueIssue` plugin name, but...